tests/test_backend.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+from keybert import KeyBERT
+from keybert.backend import SentenceTransformerBackend
+import sentence_transformers
+
+from sklearn.feature_extraction.text import CountVectorizer
+from .utils import get_test_data
+
+
+doc_one, doc_two = get_test_data()
+
+
+@pytest.mark.parametrize("keyphrase_length", [(1, i + 1) for i in range(5)])
+@pytest.mark.parametrize(
+    "vectorizer", [None, CountVectorizer(ngram_range=(1, 1), stop_words="english")]
+)
+def test_single_doc_sentence_transformer_backend(keyphrase_length, vectorizer):
+    """Test whether the keywords are correctly extracted"""
+    top_n = 5
+
+    st_model = sentence_transformers.SentenceTransformer(
+        "all-MiniLM-L6-v2", device="cpu"
+    )
```

Comment:
Could you also test whether the correct model is chosen? By default, KeyBERT uses `"all-MiniLM-L6-v2"` so by choosing the same model, you cannot test whether your method is actually working or not.